src/Beginning Visual C# Karli Watson/Chapter 3/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Chapter_3
+{
+    enum UserControl { Exercise1 = 1, Exercise2, Exercise3, Exercise4, Exercise5, Exit }
+    class Program
+    { 
+        static void Main(string[] args)
+        {
+            bool isExit = false;
+            int input;
+            const int firstControl = (int)UserControl.Exercise1;
```

Comment:
have any sense the "enum UserControl" with "const int firstControl, ..., ..."? use only one approach...